README.md

```diff
@@ -91,6 +91,24 @@ As you noticed you can set expiration time for jobs globally by expiration optio
 + It is advised to set this expiration time greater than time required for completion of the job.
 + Default expiration time is 30 minutes.
 
+### Disabling recording of the initial :queued status and metadata
+By default, the client middleware records the initial metadata at job-creation.
+If this is not desired for all workers, it can be conditionally enabled
+by setting the `record_initial_status?` method on the worker's eigenclass:
```

Comment:
Is there a difference between eigenclass and singleton class? I believe the latter is more common terminology

Reply:
from what I can tell they are used interchangeably, with metaclass as a less commonly used alternative

eigenclass seems to be the original ruby terminology

but rails/active_support used singleton_class so maybe that has become more popular?

https://en.wikipedia.org/wiki/Metaclass#In_Ruby

Reply:
`singleton_class` is a method in `Object`. I might be wrong but I think is way more common among Rubyists and in documentation is important to be clear and simple